examples/dd.qasm

```diff
@@ -23,4 +24,4 @@ box {
   cx %2, %3;
   cx %1, %2;
   u %3;
```

Comment:
`u` is not a built in gate. This needs to be replaced by `U(theta, phi, lambda)` or defined above.